modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -73,7 +73,7 @@ spec:
       rootCA: ""
 ----
 
-* Optional: Customize the `spec.sso.keycloak` field to add the route name for the `keycloak` provider in the `ArgoCD` CR. Use this feature to support advanced routing use cases, such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller shards].
+* Optional: Customize the `spec.sso.keycloak` field to add the route name for the `keycloak` provider in the `ArgoCD` CR. Use this feature to support advanced routing use cases, such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-ingress-sharding_configuring-ingress-cluster-traffic-ingress-controller[Ingress Controller shards].
```

Comment:
```suggestion
* Optional: Customize the `spec.sso.keycloak` field to add the route name for the `keycloak` provider in the `ArgoCD` CR. Use this feature to support advanced routing use cases, such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-ingress-sharding_configuring-ingress-cluster-traffic-ingress-controller[Ingress Controller sharding].
```

To match with the actual title.

Reply:
@Dhruv-Soni11 the suggested changes by @sr1kar99 above are not in scope of this PR, but I think this needs fixing. Also I would suggested changing the link to point to docs.redhat.com instead since we are decommissioning docs.openshift.com soon.

Reply:
Fixed.